lib/read.js

```diff
@@ -1,29 +1,36 @@
+'use strict';
+
 var _ = require('underscore');
 var db = require('./db');
 var SQLFilter = require('./sql_filter');
 var FilterJSCompiler = require('juttle/lib/compiler/filters/filter-js-compiler');
-var Juttle = require('juttle/lib/runtime').Juttle;
 var JuttleMoment = require('juttle/lib/moment').JuttleMoment;
 var Promise = require('bluebird');
+var AdapterRead = require('juttle/lib/runtime/adapter-read');
+var errors = require('juttle/lib/errors');
+
 
 var logger = require('juttle/lib/logger').getLogger('sql-db-read');
```

Comment:
The AdapterRead class already has a logger, so I think you could just use that one.